hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTimeline.java

```diff
@@ -70,6 +71,8 @@
   String INFLIGHT_RESTORE_EXTENSION = "." + RESTORE_ACTION + INFLIGHT_EXTENSION;
   String RESTORE_EXTENSION = "." + RESTORE_ACTION;
 
+  HoodieInstant INVALID_INSTANT = new HoodieInstant(State.INVALID, "INVALID", "0");
```

Comment:
What is the use of this ?

Reply:
come up with constants for other action and instant time too?